webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridSelection/useGridSelectionContext.tsx

```diff
@@ -238,6 +238,10 @@ export function useGridSelectionContext(tableData: ITableData, selectionAction:
     }
   }
 
+  function hasSelectedInRow(rowIdx: number) {
+    return props.selectionAction.getFocusedElement()?.row.index === rowIdx;
+  }
```

Comment:
please rename selected to focused, also seems like it's better to expose computed like focused element and in cell / header renderers use this information to detect if current cell / header is focused.
Also please note, that `selectionAction.getFocusedElement()`'s column / row is not the same as `column.idx` / `rowIdx` index in data grid you need to use cell (https://github.com/dbeaver/cloudbeaver/pull/2952/files#diff-064428d02a72e78faffcf66fbd166bc59e9692a8fb55ccf92565cc00ef13a62cR43)